otel-extension/src/main/java/io/scalac/mesmer/otelextension/MesmerAkkaActorInstrumentationModule.java

```diff
@@ -16,4 +23,83 @@ public MesmerAkkaActorInstrumentationModule() {
   public List<TypeInstrumentation> typeInstrumentations() {
     return AkkaActorAgent.agent().asOtelTypeInstrumentations();
   }
+
+  @Override
+  public Map<String, ClassRef> getMuzzleReferences() {
+    return Collections.emptyMap();
+  }
+
+  @Override
+  public void registerMuzzleVirtualFields(VirtualFieldMappingsBuilder builder) {
+    builder
+        .register("akka.dispatch.Envelope", "io.scalac.mesmer.core.util.Timestamp")
+        .register("akka.actor.ActorContext", "io.scalac.mesmer.core.actor.ActorCellMetrics");
+  }
+
+  @Override
+  public List<String> getMuzzleHelperClassNames() {
+    return Collections.emptyList();
+  }
+
+  @Override
+  public List<String> getAdditionalHelperClassNames() {
+    return Arrays.asList(
+        "io.scalac.mesmer.otelextension.instrumentations.akka.actor.impl.ActorCellDroppedMessagesAdvice",
```

Comment:
For Advice classes we need only those ending with `$` - the rest is just a template used by bytebuddy and there is no need to inject it.